utils/wasm-builder/src/wasm_project.rs

```diff
@@ -171,11 +176,13 @@ impl WasmProject {
         fs::write(
             &wasm_binary_rs,
             format!(
-                r#"
-                    pub const WASM_BINARY: &[u8] = include_bytes!("{}");
-                    pub const WASM_BINARY_OPT: &[u8] = include_bytes!("{}");
-                    pub const WASM_BINARY_META: &[u8] = include_bytes!("{}");
-                "#,
+                r#"#[allow(dead_code)]
```

Comment:
I prefer `#[allow(unused)]` here, because this code isn't dead actually

Reply:
Agree, fixed.